bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/netutils/WebSocketConnection.java

```diff
@@ -51,29 +52,33 @@
     private final Logger logger = LoggerFactory.getLogger(WebSocketConnection.class);
 
     private final WebSocketClient client;
+    private final String socketName;
+    private final Gson gson;
+
     private final WebSocketConnectionListener connectionListener;
     private final Map<Map.Entry<ResourceType, String>, WebSocketMessageListener> listeners = new ConcurrentHashMap<>();
 
-    private final Gson gson;
-    private boolean connected = false;
+    private ConnectionState connected = ConnectionState.DISCONNECTED;
```

Comment:
Just a minor suggestion.
```suggestion
    private ConnectionState connectionState = ConnectionState.DISCONNECTED;
```